wai-handler-hal.cabal

```diff
@@ -41,14 +41,14 @@ common opts
 
 common deps
   build-depends:
-    , base                  >=4.12     && <4.18
+    , base                  >=4.12     && <4.19
     , base64-bytestring     >=1.0.0.0  && <1.3
-    , bytestring            >=0.10.8   && <0.12
+    , bytestring            >=0.10.8   && <0.13
     , case-insensitive      ^>=1.2.0.0
     , hal                   >=0.4.7    && <0.4.11 || >=1.0.0 && <1.1
     , http-types            ^>=0.12.3
     , network               >=2.8.0.0  && <3.2
-    , text                  ^>=1.2.3 || ^>=2.0
+    , text                  ^>=1.2.3 || ^>=2.1
```

Comment:
This should be `text ^>=1.2.3 || ^>=2.0 || ^>=2.1` or `text ^>= 1.2.3 || >=2.0 && <2.2` depending on what the autoformatter does with it.

Reply:
I tried this but then building `aeson` failed. I'm going to leave it for now.